pkg/util/constant.go

```diff
@@ -41,6 +41,8 @@ const (
 	OpenEBSCasTypeKeySc = "cas-type"
 	// CstorCasType cas type name
 	CstorCasType = "cstor"
+	// ZFSCasType cas type name
+	ZFSCasType = "zfslocalpv"
```

Comment:
Do we need a confirmation with the naming of the cas-type flags or `zfslocalpv` is acceptable by everyone?